src/std_bridge/result.rs

```diff
@@ -44,12 +44,3 @@ pub struct ResultPtrAndPtr {
     pub is_ok: bool,
     pub ok_or_err: *mut std::ffi::c_void,
 }
-
```

Comment:
Need to also delete the C support https://github.com/chinedufn/swift-bridge/blob/27d7be991141586fb120e4691b7bea7d9eeecd17/crates/swift-bridge-build/src/generate_core/result_support.rs#L42-L44

Reply:
https://github.com/chinedufn/swift-bridge/blob/27d7be991141586fb120e4691b7bea7d9eeecd17/crates/swift-bridge-build/src/generate_core.rs#L38-L39